tests/containers/bats/podman.pm

```diff
@@ -101,7 +79,7 @@ sub run {
     record_info("podman rootless", script_output("podman info"));
 
     # Download podman sources
-    my $podman_version = get_podman_version();
+    my $podman_version = script_output "podman --version | awk '{ print \$3 }'";
```

Comment:
why? :thinking:

Reply:
> why? 🤔

For 2 reasons:

1. I want to move this source download sources to the library too, and all packages but 2 (aardvark-dns & netavark) use the 3rd field.
2. This module is the only consumer of this function.  Now we can get rid of both `get_docker_version` & `get_podman_version` in `containers::utils`.

Reply:
Fully agree then!